docker/env/hydra.sh

```diff
@@ -25,8 +26,32 @@ SCT_ARGUMENTS=()
 
 while [[ $# -gt 0 ]]; do
     case $1 in
+        --execute-on-new-runner)
+            if [[ -f "${SCT_RUNNER_IP_FILE}" ]]; then
+                SCT_RUNNER_IP=$(<"${SCT_RUNNER_IP_FILE}")
+                echo "Looks like there is another SCT runner launched already (Public IP: ${SCT_RUNNER_IP})"
+                echo "Please, delete '${SCT_RUNNER_IP_FILE}' file first and try again."
+                echo "Or use 'hydra --execute-on-runner ${SCT_RUNNER_IP} ...' to run command on existing runner"
+                exit 1
+            fi
+            echo ">>> Create a new SCT runner instance"
+            echo
+            $0 create-runner-instance \
+              --cloud-provider aws \
+              --region ${RUNNER_REGION:-us-east-1} \
+              --availability-zone ${RUNNER_AZ:-a} \
+              --test-id $SCT_TEST_ID \
+              --duration ${RUNNER_DURATION:-1440}
+            SCT_RUNNER_IP=$(<"${SCT_RUNNER_IP_FILE}")
+
```

Comment:
if [ -z "$SCT_RUNNER_IP" ]; then
    echo "SCT runner was not created"
    exit 1
fi

Reply:
Don't need this, because `hydra create-runner-instance` will exit with non-zero code and because we run script with "set -e" it will fail the whole script too.